src/shared.ts

```diff
@@ -2,13 +2,22 @@ import type { CSpellSettings, Issue } from 'cspell';
 
 // This file is created to prevent circular dependencies
 export const enum Action {
-	Ignore,
-	IgnoreAll,
-	Replace,
-	ReplaceAll,
-	SkipFile,
-	UndoLastAction,
-	Quit
+	Ignore = 'ignore',
+	IgnoreAll = 'ignore-all',
+	OpenHistory = 'open-history',
+	Quit = 'quit',
+	Replace = 'replace',
+	ReplaceAll = 'replace-all',
+	SkipFile = 'skip-file',
+	UndoLastAction = 'undo-last-action'
```

Comment:
Why did you add string values?

Reply:
Was easier to debug when logging 😬 I'll remove this on final commits

Reply:
Ok, no problem :)